lib/backburner/queue.ts

```diff
@@ -82,7 +82,7 @@ export default class Queue {
       errorRecordedForStack = queueItems[i + 3]; // Debugging assistance
 
       // method could have been nullified / canceled during flush
-      if (method) {
```

Comment:
`method` could be either function or null and for other cases I think it should break because bb shouldn't allow other cases to pass through to queue, and `typeof method === 'function'` being slower than `method !== null` I think it is overkill that can be avoided